packages/entities/entities-shared/package.json

```diff
@@ -0,0 +1,74 @@
+{
+  "name": "@kong-ui-public/entities-shared",
+  "version": "1.0.0",
+  "type": "module",
+  "main": "./dist/entities-shared.umd.js",
+  "module": "./dist/entities-shared.es.js",
+  "types": "dist/types/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "exports": {
+    ".": {
+      "import": "./dist/entities-shared.es.js",
+      "require": "./dist/entities-shared.umd.js"
+    },
+    "./package.json": "./package.json",
+    "./dist/*": "./dist/*"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "peerDependencies": {
+    "@kong-ui-public/i18n": "workspace:^",
```

Comment:
I believe should have a version in the entry

Reply:
This is actually what I got after running `pnpm --filter "@kong-ui-public/entities-shared" add --save-peer @kong-ui-public/i18n@latest`. Plus, I think this [solves dependency conflicts](https://kongstrong.slack.com/archives/C03ANC23AQK/p1680581451818809?thread_ts=1680529934.588149&cid=C03ANC23AQK).

Reply:
Easy way to check is to run the build, then `npm pack` and check the outputted file to ensure it has the correct resolved version

Reply:
<img width="351" alt="image" src="https://github.com/Kong/public-ui-components/assets/10095631/9a6542d0-33a6-493e-a45b-2d2dee31f03f">

Yes it has.